examples/official-storybook/stories/addon-storyshots.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import BaseButton from '../components/BaseButton';
+
+const text = 'Testing the storyshots addon';
+
+storiesOf('Addons|Storyshots', module)
+  .add('Default', () => <BaseButton label="" />)
```

Comment:
Using an an unstyled button for those stories isn't a really good idea, because it uses appearance of system UI button, which means the differences between OS are huge

E. g., this is how [storyshots story](https://deploy-preview-3102--storybooks-official.netlify.com/?selectedKind=Addons%7CStoryshots&selectedStory=Label&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel) looks on mac
<img width="195" alt="screen shot 2018-03-09 at 01 41 03" src="https://user-images.githubusercontent.com/6651625/37182651-fdb02c88-233a-11e8-90ed-8c3213362aad.png">

Reply:
Oh so true ! I'll change the test, did a bit too quickly.